middleware/response_header.go

```diff
@@ -0,0 +1,22 @@
+package middleware
+
+import (
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/brave/go-sync/utils"
+)
+
+const (
+	saneTimeMillsHeaderKey = "Sane-Time-Millis"
+)
+
+// ResponseHeader is a middleware to apply common response headers.
+func ResponseHeader(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I would recommend that this be named `CommonResponseHeaders` or something, and the middleware would iterate over a list of header keys/functions which add particular headers, so you can add more easily:

```go
func saneTimeMillsHeaderFunc(w http.ResponseWriter) {
    w.Header().Set("Sane-Time-Millis", strconv.FormatInt(utils.UnixMilli(time.Now()), 10))
}
var (
    headerFuncs = []func(w http.ResponseWriter) {
        saneTimeMillsHeaderFunc,
    } 
)
```

Then inside here you could range over the header funcs.